emscripten.py

```diff
@@ -2112,6 +2112,7 @@ def emscript_wasm_backend(infile, outfile, memfile, libraries, compiler_engine,
     t = time.time()
 
   forwarded_json = json.loads(forwarded_data)
+  assert not forwarded_json['Functions']['implementedFunctions']
```

Comment:
Maybe a helpful assert message? Why do we expect there to be an `implementedFunctions` key, and why is it empty?

Similarly, can we change this to `forwarded_json['Functions'].get('implementedFunctions', [])`, so that we can asynchronously remove that key from binaryen's output in the future?

Reply:
This is coming from compiler.js.. not from binaryen.  Its a little confusing.  Added a comment.

Reply:
Oof, yeah good call.